frontend-calculator/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "frontend-calculator",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "gen-graphql": "graphql-codegen --config codegen.yml"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.6.9",
+    "@cartesi/rollups": "^0.4.0",
+    "@ethersproject/providers": "^5.5.3",
+    "@types/styled-components": "^5.1.25",
+    "@urql/core": "^2.4.3",
```

Comment:
The current version of our React Apps don't use urql package anymore. You can see the current packages [here](https://github.com/OsmanRodrigues/rollups-examples/blob/dev/frontend-m2cgen/package.json)